### PR TITLE
add new pyelastisearch version

### PR DESCRIPTION
**Pull request by nabilalibou**

### CODE REVIEW

**Review by matz-e — COMMENTED** [review]
This should be done with `spack augment` and end up in `patches` (the version mod only)

Otherwise there's a risk that this will be dropped on the next merge from upstream.

**Comment by nabilalibou** [comment]
> This should be done with `spack augment` and end up in `patches` (the version mod only)
> 
> Otherwise there's a risk that this will be dropped on the next merge from upstream.

Yes I just remembered this, i pushed a patch

File: bluebrain/repo-patches/packages/py-elasticsearch/package.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from spack.pkg.builtin.py_elasticsearch import PyElasticSearch as BuiltinPyElasticSearch
+
+
+class PyElasticSearch(BuiltinPyElasticSearch):
+    __doc__ = BuiltinPyElasticSearch.__doc__
+
+    version('7.6.0', sha256='d228b2d37ac0865f7631335268172dbdaa426adec1da3ed006dddf05134f89c8')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-urllib3@1.8:1', type=('build', 'run'))
```
Comment (matz-e) [inline bluebrain/repo-patches/packages/py-elasticsearch/package.py]:
```suggestion
```
Dependencies should be inherited from the builtin :)

**Comment by nabilalibou** [comment]
thanks for the commit (i just saw it and was wondering why i could not push these changes myself but needed to refresh haha).
I'll use augment next time, less mistakes even if you need to go through .py modification with terminal

**Comment by matz-e** [comment]
> thanks for the commit (i just saw it and was wondering why i could not push these changes myself but needed to refresh haha)

sorry, thought this was quicker, and I was just in my Spack checkout anyways :)

**Review by matz-e — APPROVED** [review]
I _think_ this should work now (famous last words)

**Comment by nabilalibou** [comment]
> I _think_ this should work now (famous last words)

Allowing auto-merge has reset the approved review I think

Edit: nevermind I though that the pending test was your missing review (because it has your profile icon)

**Pull request merged**